requirements.min.txt

```diff
@@ -15,4 +12,3 @@ pympi-ling==1.69
 python-slugify==5.0.0
 soundswallower~=0.4.1
 webvtt-py==0.4.2
-werkzeug==2.0.3
```

Comment:
hooray :)

Reply:
Don't celebrate too fast, that dependency is still in g2p...